README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# HW1
+
+# About me
+My name is **Chenyang He**. I am a MAPSS student concentrating in *Psychology*. 
+
+# Research Interest 
+I am currently working as an RA in Dr. Boaz Keysar's Multilingualism and Decision-Making Lab in the Psychology Department. 
+Here's a link to our lab page:
+[mdl](https://mdl.uchicago.edu/)
```

Comment:
I don't have a lot of suggestions, but on a stylistic note, it might be interesting to incorporate the link into the description. Ie make the "Multingualism and Decision Making Lab" in the previous sentence itself the hyperlink, so you don't have to add "here's a link to our lab page"